docs/source/gallery/utils/01_add_reference_bands.py

```diff
@@ -17,18 +17,17 @@
 azp.style.use("arviz-variat")
 
 data = load_arviz_data("centered_eight")
-references = [(0, 5), (5, 20)]
-pc = azp.plot_dist(
+rope = [(-1, 1)]
+pc = azp.plot_forest(
     data,
-    kind="ecdf",
     backend="none",   # change to preferred backend
 )
-
-pc = azp.add_reference_bands(
+pc.coords = {"column": "forest"}
+pc = azp.add_bands(
     pc,
-    references=references,
+    values=rope,
     aes_map={"ref_band": ["color"]},
-    color=["black", "gray"]
+    color=["#f66d7f"]
```

Comment:
doesn't using `plot_kwargs` work? It would be important to know why not if it doesn't, and conceptually I think it is what we want here (even if 1 element aes mapping achieves the same result)

Reply:
yes, it does.